src/components/VideoList/VideoItem/VideoItem.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const VideoItemStyled = styled.div`
+  flex-direction: column;
+  flex-flow: row wrap;
+  border-radius: 5px;
+  box-shadow: 0px 2px 1px -1px rgb(0 0 0 / 20%), 0px 1px 1px 0px rgb(0 0 0 / 14%),
+    0px 1px 3px 0px rgb(0 0 0 / 12%);
+  width: 345px;
+  height: 345px;
+  margin: 10px;
+  overflow: hidden;
+  display: inline-block;
+  background-color: #fff;
+  transition: 0.3s;
+  & .content {
+    padding: 16px;
+  }
+  & div,
+  & h2,
+  & p {
+    margin: 0px;
+    padding: 0px;
+  }
+  & h2 {
+    font-size: 20px;
+    text-align: left;
+    color: #333;
+    font-weight: normal;
+    margin-bottom: 0.35em;
+  }
+  & p {
+    font-size: 14px;
+    text-align: left;
+    color: rgba(0, 0, 0, 0.54);
+    font-weight: normal;
+  }
+  & .imageMask {
```

Comment:
Another approach would be to separate this into another component (you can define it in this same file since it's only going to be used here). I don't have much problem with this approach but the styling grow a lot.